cmd/juju/firewall/listrules.go

```diff
@@ -56,7 +56,6 @@ func (c *listFirewallRulesCommand) Info() *cmd.Info {
 		Name:    "list-firewall-rules",
 		Purpose: listRulesHelpSummary,
 		Doc:     listRulesHelpDetails,
-		Aliases: []string{"firewall-rules"},
```

Comment:
Do we want to keep `firewall-rules` and remove `list-firewall-rules`? That way we'll be consistent with other commands

Reply:
I see no reason why we should have `list-firewall-rules` as an exception so I've decided to go ahead with this